compiler/rustc_transmute/src/maybe_transmutable/mod.rs

```diff
@@ -44,18 +44,18 @@ mod rustc {
             let Self { src, dst, assume, context } = self;
 
             let layout_cx = LayoutCx { tcx: context, param_env: ParamEnv::reveal_all() };
-            let layout_of = |ty| {
+            /*let layout_of = |ty| {
```

Comment:
Delete this commented out code please.

Reply:
D'oh, [done](https://github.com/rust-lang/rust/compare/dff0f993e3659c7e0a4e0957f405f2fcba5c7980..a0fa0e014c0f7afbcac21e4313014b871ba1e250)!